config/config.go

```diff
@@ -408,9 +428,20 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if len(c.Route.Match) > 0 || len(c.Route.MatchRE) > 0 {
 		return fmt.Errorf("root route must not have any matchers")
 	}
+	if len(c.Route.MuteTimes) > 0 {
+		return fmt.Errorf("root route cannot have any mute times")
```

Comment:
we should be consistent and pick a unique name

mute times
mute times interval ?

Reply:
This is not addressed. Also we use 'must not' above